### DISCUSSION TITLE
Achifments

### DISCUSSION THREAD

**Discussion by banderas328** [community/community#18904] [Other Feature Feedback, Questions, & Ideas]
Thanks for the achifments block. It's nice and cute.

**Comment by melvincwng** [comment]
Hi @banderas328, for a full list of achievements that can be obtained, you can refer to this list here for more info:

https://github.com/Schweinepriester/github-profile-achievements

Happy achievement hunting :)

**Comment by thuongtruong1009** [comment]
Hi @banderas328 
* Introducing Achievements is recognizing the many stages of a developer’s coding journey. So have full list instructions about how to collect it at https://github.com/Schweinepriester/github-profile-achievements
* In the future, you can easily share your achievements badge on social platforms with friends. Following at https://github.blog/2022-06-09-introducing-achievements-recognizing-the-many-stages-of-a-developers-coding-journey/
* Besides, Github also update the interface for the 3D skyline contribution graph https://skyline.github.com/. They are cool, right!😊

##### Hope it is useful for you and have a nice day! 🥰
